.github/file-paths.yaml

```diff
@@ -0,0 +1,69 @@
+shared_sources: &shared_sources
+  - "shared/src"
+
+shared_specs: &shared_specs
+  - "shared/test"
+
+frontend_sources: &frontend_sources
+  - *shared_sources
+  - "frontend/**"
+  - "enterprise/frontend/**"
+  - "frontend/!(test)/**"
+  - "yarn.lock"
+  - "**/tsconfig*.json"
+  - "package.json"
+  - "babel.config.json"
+  - "postcss.config.js"
+  - "webpack.config.js"
+  - "webpack.static-viz.config.js"
+
+frontend_specs: &frontend_specs
+  - *shared_specs
+  - "frontend/test/!(__support__|__runner__)/**"
+  - "frontend/**.unit.*"
+  - "jest.config.js"
+
+frontend_all: &frontend_all
+  - *frontend_sources
+  - *frontend_specs
+
+backend_presto_kerberos:
+  - '**/presto_jdbc/**'
+  - '**/presto_jdbc.clj'
+
+backend_sources: &backend_sources
+  - *shared_sources
+  - "enterprise/backend/src/**"
+  - "src/**"
+  - "*modules/**"
```

Comment:
I think we have to explicitly add `deps.edn` as it is the most important config file for BE.

Reply:
@qnkhuat suggested it's safer to include even nested deps. `**deps.edn`

Reply:
Good catch, thanks. I added this path